### ISSUE TITLE
This faceRecognition is how different than your opencv4nodejs faceRecognition?

### ISSUE THREAD

**Issue by komalpharate** [justadudewhohacks/face-recognition.js#88]

**Comment by justadudewhohacks** [comment]
face-recognition.js is a js wrapper around face detection and recognition stuff from dlib. The FaceRecognizerNet works with a neural net under the hood, which is based on a resnet 34 architecture and uses a pretrained model, provided by the maintainer of dlib, which achieves ~99% accuracy on the LFW benchmark for facial recognition for 150x150 sized face images.

OpenCV FaceRecognizers are not based on deep learning, they implement other algorithms, which are no as accurarate and not very scalable.
